package.json

```diff
@@ -2,22 +2,606 @@
   "name": "leaderboard",
   "version": "1.0.0",
   "description": "The leaderboard website displays scores submitted by different players",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Nice work configuring Webpack command for production mode (`"build": "webpack"`) :clap: However the command for building Webpack in development mode is not configured (`"start": "webpack serve --open"`). Kindly make sure it's configured and the command `npm run start` or `npm run dev` run the **dev server**.